testdata/cast.sql

```diff
@@ -0,0 +1,11 @@
+-- safe: int is automatically interpreted as pg_catalog.int
+SELECT 1::int;
+
+-- safe: int4 is the built-in pg_catalog.int4
```

Comment:
I think that this should be safe, but currently pgspot is warning about it (see `PS017: Unqualified object reference: int4 in CAST(1 AS int4)`).

Reply:
Yes, thank you. I had a misunderstanding of how `pg_catalog` is handled in `search_path`.

Reply:
done